src/css/popup.css

```diff
@@ -783,8 +792,20 @@ span ~ .panel-header-text {
   padding-inline-start: 16px;
 }
 
+/**
+ Bugfix: incorrect scrollbars/spacing when editing container with many assignments
+ https://github.com/mozilla/multi-account-containers/issues/948
+ */
+.edit-container-panel .columns {
+  overflow: hidden;
+}
+
 #edit-sites-assigned {
-  flex: 1;
+  /**
+   Bugfix: incorrect scrollbars/spacing when editing container with many assignments
+   https://github.com/mozilla/multi-account-containers/issues/948
+   */
+  flex: 1000;
```

Comment:
Can you explain why the 1000 value, it seems like a code smell from what I can tell?

Reply:
The value needs to be bigger than 1, so that the assignments part of the panel grows to fill the available space in preference to the top part of the panel (containing the name, color, icon). Exactly how much bigger than 1 it needs to be is debatable. Possibly 2 would have been sufficient. But I chose 1000 so that it is obvious that this part of the panel must take ALL free space, and not share even the slightest bit of it.

Hopefully the below pictures will give you an idea of the issue. The first picture shows the existing bug. The last picture shows my solution, using `flex: 1000`. You can see it's identical to the picture that uses `flex: 2`. Personally, I would stick with 1000.

As one final side note, usually you would use `flex: 0` and `flex: 1` to control what grows to fill empty space, and what doesn't. But this isn't possible in this situation, because the panel for adding a new container shares the same HTML as this one for editing an existing container, and although `flex: 0` and `flex: 1` works for the latter (edit container), it messes up the spacing on the former (add container).

<img width="306" alt="flex-1" src="https://user-images.githubusercontent.com/439593/65461972-1eb0cb00-de55-11e9-9b73-1f0e56b324ce.png">

<img width="305" alt="flex-1 2" src="https://user-images.githubusercontent.com/439593/65461977-23757f00-de55-11e9-8d9b-76cb1a8d7b58.png">

<img width="305" alt="flex-1 5" src="https://user-images.githubusercontent.com/439593/65461981-26706f80-de55-11e9-9f10-f736dff599a2.png">

<img width="301" alt="flex-2" src="https://user-images.githubusercontent.com/439593/65461987-2a03f680-de55-11e9-9754-6ba0096a2d1f.png">

<img width="303" alt="flex-1000" src="https://user-images.githubusercontent.com/439593/65461995-2cfee700-de55-11e9-805f-7bf7a3acb4ba.png">

Reply:
Cool thanks for the explanation, thats good enough for me. We could have overriden the flex value for the top panel with a parent class I guess but it's debatable if it's worth it.